### DISCUSSION TITLE
Possible problem in file org/eclipse/jface/text/contentassist/CompletionProposalPopup.java#L1248

### DISCUSSION THREAD

**Discussion by petrufm** [eclipse-platform/eclipse.platform.ui#1184] [General]
Hi,

In file https://github.com/eclipse-platform/eclipse.platform.ui/blob/master/bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/contentassist/CompletionProposalPopup.java#L1248 I observed it would be possible to make visible a content assist without arming its CompletionProposalPopup (this) listener. Is this intended?  Among others, multiple content assists may be made visible simultaneously.

Thanks.

**Comment by mickaelistria** [comment]
If you can provide steps to reproduce the issue, please open an issue report with those.

**Reply by petrufm** [reply]
Done (https://github.com/eclipse-platform/eclipse.platform.ui/issues/1185#issue-1923476738).
